libstorage/src/SystemCmd.cc

```diff
@@ -256,6 +288,11 @@ SystemCmd::doExecute( string Cmd )
 		    y2error( "dup2 stderr child failed errno=%d (%s)", errno,
 			     strerror(errno));
 		    }
+		if( close( sin[1] )<0 )
+		    {
+		    y2error( "close child failed errno=%d (%s)", errno,
```

Comment:
Right, but I wanted to keep it as similar to the surrounding code as possible -- and there it's similarly impossible to tell which close() went wrong - stdout or stderr? Now it's one more to guess from. But when that happens, something went badly wrong already, so it's just cosmetic which of those close() calls failed.